server/settings/main.py

```diff
@@ -301,6 +301,11 @@ class CoreSettings(BaseSettingsModel):
         "{}",
         widget="textarea",
         title="Project folder structure",
+        description=(
+            "Defines project folders to create on 'Create project folders'."
+            " When empty, the action will be hidden from users in the"
+            " launcher because there is nothing to create."
+        ),
```

Comment:
I understand it will create the project structure in the setting. 
but, could we be clear that it is an additional structure that is not necessarily related to the project structure in the project editor?

Reply:
Do you mean like adding "This does not create folder and task entities, but creates project folders on disk unrelated to any entity"? It seems quite long and verbose and still unclear too. Any ideas on how to explain it with less words and more correct?

Reply:
BTW it is still used for actions that actually do use this setting to create it for project folder structure.

Reply:
> Yes - so you're saying it will not do anything there when empty but it will still be visible/shown as action (it does not get hidden there). Correct?

No, I'm saying that this setting is used for 2 features:
1. Create folder structure in project.
2. Create folder structure on disk.

Reply:
> Create folder structure in project.

Wait - what!? Where? How? So this **does** create entities?

Reply:
Ah found it - it's here `ayon-ftrack`: https://github.com/BigRoy/ayon-ftrack/blob/288b7c6609df78f2ac283cffa55f1efed2905779/client/ayon_ftrack/event_handlers_user/action_create_project_structure.py#L88-L90

Hmm.

Reply:
Like I wrote, that was original purpose of these settings https://github.com/ynput/ayon-ftrack/blob/develop/client/ayon_ftrack/event_handlers_user/action_create_project_structure.py . It was then used to also create folders on disk.

Reply:
That's why the "disk" word is important for the button name.
With "Create Project Folders", people will think it creates folder entities.

Reply:
So this: https://github.com/ynput/ayon-core/pull/679/commits/5065e0e8dbb4da9e4a05918abced984b2f077e5d

![image](https://github.com/user-attachments/assets/1b0afb17-207e-4a56-b532-e38a5ce1860b)

No idea how it formats it like this haha 🤦‍♂️

Reply:
> That's why the "disk" word is important for the button name.
> With "Create Project Folders", people will think it creates folder entities, and they will be surprised to discover it created disk structure.

@dee-ynput ? :)

Reply:
> Do you mean like adding "This does not create folder and task entities, but creates project folders on disk unrelated to any entity"? It seems quite long and verbose and still unclear too. Any ideas on how to explain it with less words and more correct?

I mean something like this. 
imo, we may not add all of the that to the tool tip in favor of adding them to the documentation. 

```
Defines project folders to be created when selecting 'Create project folders'. 

- If you use the exact same structure in the project editor, it will create all the folders.
- If you use a different structure, it will add them as additional directories.
- If the setting is empty, the 'Create project directories' action will be hidden from users in the launcher.

This setting is also used in ftrack addon to create both the folders on disk and the ftrack entities.
```

Reply:
> - If you use the exact same structure in the project editor, it will create all the folders.
> - If you use a different structure, it will add them as additional directories.

I personally don't understand these? Same structure to what? :) 

Anyway, make a decision and I'll be happy to adjust accordingly - so we don't keep this lingering much longer.

Reply:
I think the new tool tip is much better.